flipperdevices/Flipper-Android-App
# Remotecontrols/dialogs

**Background**

On new remote-controls screen we don't have much dialogs to determine current flipper conditions. This PR adds new dialogs to improve the situation

**Changes**

- Add dialogs into remote-controls for NotSupported/Busy/NotConnected
- Move remotecontrols into bottombar

**Test plan**

- Open some saved remote or new remote with flipper disconnected or busy
- try press button and see dialog
